CHANGELOG.rst

```diff
@@ -106,10 +141,6 @@ Changed
 * New Attribute ``dbVersion`` is included in the GET Response for Autonomous Transaction Processing Database and Autonomous Data Warehouse.
 * New Attribute ``allConnectionStrings`` is included in the GET Response for Autonomous Transaction Processing Database and Autonomous Data Warehouse.
 
-Known Issues
```

Comment:
Instead of removing this, we should add a line to our latest changelog saying this has been fixed

Reply:
The rationale was that fixing this was a server side change so we wanted to remove it from 'Known Issues' so future users of this version won't think that it is a limitation of this specific CLI version.